packages/ndla-ui/src/Article/ArticleNotions.tsx

```diff
@@ -0,0 +1,244 @@
+    position: fixed;
+    border: none;
+    padding: 0;
+    height: auto;
+    top: 12rem;
+    right: 18px;
+    background-color: transparent;
+    z-index: 10;
+  }
+
+  ${mq.range({ from: '1024px' })} {
+    right: calc(50vw - 1024px / 2 + 19px);
+  }
+
+  svg {
+    &:first-of-type {
+      display: none;
+      width: 40px;
+      height: 54px;
+      cursor: pointer;
+      color: #def1ed;
+
+      ${mq.range({ from: breakpoints.tablet })} {
+        display: block;
+      }
+    }
+    &:nth-of-type(2) {
+      display: block;
+      color: ${colors.brand.primary};
+      margin-right: 8px;
+
+      ${mq.range({ from: breakpoints.tablet })} {
+        display: none;
+      }
+    }
+  }
+
+  span {
+    ${fonts.sizes('16px', '39px')};
+    display: block;
+    font-family: ${fonts.sans};
+    font-weight: ${fonts.weight.semibold};
+    color: ${colors.brand.primary};
+    padding-bottom: 1px;
+
+    ${mq.range({ from: breakpoints.tablet })} {
+      display: none;
+    }
+  }
+`;
+
+const ModalHeadingContainer = styled.div`
+  display: flex;
+  align-items: center;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  margin-bottom: 2rem;
+
+  ${mq.range({ from: breakpoints.tablet })} {
+    padding-left: 3.5rem;
+    padding-right: 3.5rem;
+  }
+
+  svg {
+    display: block;
+    width: 2rem;
+    height: 2rem;
+    color: #638b98;
+    margin: 0 1rem 0 -0.25rem;
+
+    ${mq.range({ from: breakpoints.tablet })} {
+      width: 3rem;
+      height: 3rem;
+      margin: 0 1.5rem 0 -0.5rem;
+    }
+  }
+
+  h1 {
+    margin: 0;
+  }
+`;
+
+const NotionsContainer = styled.div`
+  padding: 0 20px;
+
+  ${mq.range({ from: breakpoints.tablet })} {
+    padding: 0 3.5rem;
+  }
+`;
+
+const RelatedContentContainer = styled.ul`
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  margin: 0 0 2rem;
+  padding: 0 20px;
+
+  ${mq.range({ from: breakpoints.tablet })} {
+    padding: 0 3.5rem;
+  }
+
+  &:not(:last-child) {
+    margin-bottom: 1.5rem;
+  }
+
+  > li {
+    width: 100%;
+
+    &:not(:nth-child(2n)) {
+      margin-right: 0;
+    }
+
+    ${mq.range({ from: breakpoints.tablet })} {
+      width: calc(100% * (1 / 3));
+
+      &:not(:nth-child(2n)) {
+        margin-right: 0.5rem;
+      }
+    }
+  }
+`;
+
+export type NotionItem = {
+  id: string;
+  title: string;
+  text: string;
+  authors: {
+    name: string;
+  }[];
+  license: string;
+  image: {
+    url: string;
+    alt: string;
+  };
+  url: string;
+};
+
+export type NotionRelatedContent = {
+  url: string;
+  label: string;
+};
+
+type ArticleNotionsProps = {
+  locale: string;
+  notions: NotionItem[];
+  onReferenceClick?: React.MouseEventHandler<HTMLButtonElement>;
+  relatedContent?: NotionRelatedContent[];
+  renderMarkdown: (text: string) => string;
+};
+
+export const ArticleNotions: React.VFC<ArticleNotionsProps & tType> = ({
+  locale,
+  notions,
+  onReferenceClick,
+  relatedContent = [],
+  renderMarkdown,
+  t,
+}) => (
```

Comment:
Synes dette formatet er bedre. Då må VFC importeres sammen med react.
```suggestion
export const ArticleNotions = ({
  locale,
  notions,
  onReferenceClick,
  relatedContent = [],
  renderMarkdown,
  t,
}: VFC<ArticleNotionsProps & tType>) => (
```

Reply:
Fikset